.github/workflows/test.yml

```diff
@@ -34,3 +34,8 @@ jobs:
 
     - name: Test
       run: make test-coverage
+
+    - name: Verify Examples
+      env:
+        EXAMPLES_DIR: _examples
+      run: go test -v -run='^TestBuildExamples$'
```

Comment:
```suggestion
      run: make test
```

Reply:
Since "normal" tests are already being run as part of the coverage, isn't it better to just run the example tests for this new step? It would make the CI faster.

Reply:
The suggestion for `make test` was because that would also introduce `-race`, which we are not doing at the moment.

Reply:
I see, I have changed it to `make test`, however a test is failing. I will look into it to see why.

Reply:
Hey @crazybolillo, let's keep the previous change you had `go test -v -run='^TestBuildExamples$'`. 

The `make test` has some intermittent issues due to the `-race`. Feel free to propose a separate PR for that.

Reply:
Sorry for the late reply. I have gone back to just the build examples. Will see if I can create a separate PR to fix the flakiness from `-race`